examples/Makefile

```diff
@@ -1,36 +1,32 @@
-.PHONY: test2d fft_physical_x fft_physical_z halo_test init_test io_test 
+.PHONY: init_test test2d fft_physical_x fft_physical_z io_test 
 
 # Just build the examples
-examples: test2d fft_physical_x fft_physical_z halo_test init_test io_test
+examples: init_test test2d fft_physical_x fft_physical_z io_test
 	@echo "Built the examples"
 
+init_test:
+	$(MAKE) -C $@ all
 test2d:
 	$(MAKE) -C $@ $@
 fft_physical_x:
 	$(MAKE) -C $@ all
 fft_physical_z:
 	$(MAKE) -C $@ all
-halo_test:
```

Comment:
The test `halo_test` is removed. Why ?

Reply:
I am finishing halo. This is why i have removed it. As soon as the code is working properly i’ll add it back

Reply:
I think approach to test on atleast CPU is best. We can add GPU test later